### ISSUE TITLE
Migration instructions have an odd step. 

### ISSUE THREAD

**Issue by g7morris** [Islandora-Collaboration-Group/ISLE#419]
#### Issue description
Describe the Issue you are experiencing here.  If it's not an issue but a new feature request, please describe the new feature here.

https://islandora-collaboration-group.github.io/ISLE/install/install-staging-migrate/

Has a step for using `install_solution_packs.sh: installs Islandora solution packs` which is a destructive command on a remote. 

Please review the instructions to determine if this is an issue.

**Comment by ysuarez** [comment]
I just wanted to mention that this "install_solution_packs.sh: installs Islandora solution packs" step is not found in the "[Production Migrate](https://islandora-collaboration-group.github.io/ISLE/install/install-production-migrate/#step-16-on-remote-production-run-isle-scripts)" docs. So if this step needs to be removed, it may only need to be remove from the "Staging Migrate"

**Comment by bondjimbond** [comment]
@g7morris Can you elaborate on how it's destructive? I've followed these directions several times including the `install_solution_packs.sh` script without problems.

**Comment by ysuarez** [comment]
@bondjimbond thanks for commenting. I was the one the brought up this potential issue to Gavin and it was just based on a hunch, but he obviously understands this more than me. My thinking was that for a "migrate staging" or "migrate production" ISLE7 scenario, I would not want to make any changes to the recently copied Feora data in datastreamStore and objectStore in my staging server, and I believe `install_solution_packs.sh`does touch my migrated Fedora data. Of course the `install_solution_packs.sh`is needed when it is a "new staging" or "new production" scenario. Though it sounds like if you did not see any issues on your end when running a migrate scenario and running this command?

**Comment by bondjimbond** [comment]
> Though it sounds like if you did not see any issues on your end when running a migrate scenario and running this command?

Nope, no issues using it. It may be comforting to note that it's possible to reinstall the solution packs through the UI as well, and it's something that you might have to do once in a while. If you look at the Solution Packs Required Objects setting - admin/islandora/solution_pack_config/solution_packs - you can force reinstall them. I've done this several times on my massive (12 TB) multisite repository because something had gone wrong. There was no effect on the existing objects in the repository.

**Comment by ysuarez** [comment]
@bondjimbond thanks for all this info! I was having some issues with my ISLE7 staging install, but I already didn't think it was related to that script since I rebuilt my staging without using the script and the issue persited. I also checked for changes between my original Fedora data in the current production server and my staging data, and they were exactly the same.
